bin/list-all

```diff
@@ -20,7 +20,7 @@ sort_versions() {
 list_versions() {
   cmd="curl -fsS --url $RELEASES_URL"
   [ -n "$GITHUB_API_TOKEN" ] && cmd+=" -H 'Authorization: token $GITHUB_API_TOKEN'"
-  echo "$(eval $cmd | grep -oE "tag_name\": *\".*\"," | sed 's/tag_name\": *\"//;s/\",//')"
+  echo "$(eval $cmd | grep -oE "tag_name\": *\".*\"," | sed 's/tag_name\": *\"//;s/\",//' | sed 's/jq-//')"
```

Comment:
Rather than call sed multiple times, this can be simplified.

```suggestion
  echo "$(eval $cmd | grep -oE "tag_name\": *\".*\"," | sed 's/tag_name\": *\"//;s/\",//; s/jq-//')"
```